usrsctplib/usrsctp.h

```diff
@@ -967,6 +969,34 @@ usrsctp_deregister_address(void *);
 int
 usrsctp_set_ulpinfo(struct socket *, void *);
 
+void
+usrsctp_fire_timer(int);
+
+int
+usrsctp_open_sctp4_socket(void);
+
+int
+usrsctp_open_udpsctp4_socket(void);
+
+int
+usrsctp_open_sctp6_socket(void);
+
+int
+usrsctp_open_udpsctp6_socket(void);
+
+int
+usrsctp_recv_function_sctp4(int, struct mbuf **, struct iovec *);
+
+int
+usrsctp_recv_function_udpsctp4(int, struct mbuf **, struct iovec *);
+
+int
+usrsctp_recv_function_sctp6(int, struct mbuf **, struct iovec *);
+
+int
+usrsctp_recv_function_udpsctp6(int, struct mbuf **, struct iovec *);
+
+
```

Comment:
I think all the usrsctp_recv_function() needs to be implemented. Or am I missing something?